src/main/scala/temple/generate/kube/ast/gen/volume/AccessMode.scala

```diff
@@ -0,0 +1,6 @@
+package temple.generate.kube.ast.gen.volume
+
+object AccessMode {
+  type AccessMode = String
+  val ReadWriteMany: AccessMode = "ReadWriteMany"
```

Comment:
These don’t feel particularly typesafe, perhaps a native Enumeration would work better? Native enums `toString` to their name